app/views/users/index.html.haml

```diff
@@ -0,0 +1,25 @@
+.container
+  %h3
+    Users
+    %span.pull-right
+      = link_to 'Create User', new_user_path, :class => "btn btn-lg btn-danger", :style => 'margin: 5px; padding: 5px;'
+  .row
+    .col-md-12
+      %table.table-striped.table-bordered.table
+        %tr
+          %th Fullname
+          %th Currently
+          %th.text-center{colspan: 3} Actions
+        - @users.each do |user|
+          %tr
+            %td
+              = link_to user.fullname, user_path(user)
+            %td.status
+              = user.status
+            %td
+              = link_to 'Trim', trim_user_path(user), :data => {:action => "user_action"}
```

Comment:
Is there any significance to the ```user_action``` ?  This seems to be a wholly arbitrary string.